src/core/ext/transport/chttp2/client/secure/secure_channel_create.c

```diff
@@ -97,15 +98,15 @@ grpc_channel *grpc_secure_channel_create(grpc_channel_credentials *creds,
   GRPC_API_TRACE(
       "grpc_secure_channel_create(creds=%p, target=%s, args=%p, "
       "reserved=%p)",
-      4, (creds, target, args, reserved));
+      4, ((void *)creds, target, (void *)args, (void *)reserved));
   GPR_ASSERT(reserved == NULL);
   // Make sure security connector does not already exist in args.
```

Comment:
Fair enough. In my opinion, it does look much cleaner now. Only grpclb (its secure version anyway) and the check_peer() in the relevant security connectors need changes now. Everything else stays the same.